src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/index.md

```diff
@@ -111,6 +111,37 @@ If left open, the installation dialog waits for the agent to establish a connect
 A successful health check indicates that deployments can successfully be executed.
 :::
 
+## Upgrading the Kubernetes agent
+
+The Kubernetes agent can be upgrade by the Octopus portal or via a `helm` command.
+
+### Octopus portal
+
+To check if a Kubernetes agent can be upgraded, navigate to the **Infrastructure ➜ Deployment Targets ➜ [DEPLOYMENT TARGET] ➜ Connectivity** page. If the agent can be upgraded, there will be an *Upgrade available* banner. Clicking **Upgrade to latest** button will trigger the upgrade via a new task. If the upgrade fails, the previous version of the agent is restored.
+
+:::figure
+![Kubernetes Agent updated interface](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/kubernetes-agent-upgrade-portal.png)
+:::
+
+### Helm upgrade command
+
+To upgrade a Kubernetes agent via `helm`, note the following fields from the **Infrastructure ➜ Deployment Targets ➜ [DEPLOYMENT TARGET] ➜ Connectivity** page:
+* Helm Release Name
+* Namespace
+
+Then, from a terminal connected to the cluster containing the instance, execute the following command (replacing the release name + namespace noted above)
+
+```
```

Comment:
I think if you specify the language then you get the "copy" icon

like here: https://octopus.com/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/storage#KubernetesAgentStorage-Step2-ChangePersistence

Reply:
done